### DISCUSSION TITLE
submodule packages/resources.binary path is gz-tea.ejoy.com

### DISCUSSION THREAD

**Discussion by ylme** [ejoy/ant#13] [General]
目前 packages/resources.binary 的引用的仓库是内网的 https://gz-teain.ejoy.com/git/Ant/assets.git 链接，正常来说这个是要去掉的吧。

**Comment by cloudwu** [comment]
是一些测试用的资源，不确定还有没有用。权限可以下周开。

**Reply by cloudwu** [reply]
已经把这个子模块移到 github 的公开仓库了。
